include/lsst/jointcal/AstrometryFit.h

```diff
@@ -52,13 +52,10 @@ class Associations;
  * CcdImage's. One does not need reference object and would then naturally not
  * have any Kind 2 terms.
  */
-class AstrometryFit {
+class AstrometryFit : public FitterBase {
 private:
-    Associations &_assoc;
-    std::string _whatToFit;
     bool _fittingDistortions, _fittingPos, _fittingRefrac, _fittingPM;
-    AstrometryModel *_astrometryModel;
-    int _LastNTrip;                   // last triplet count, used to speed up allocation
+    AstrometryModel &_astrometryModel;
```

Comment:
I've deleted the move/copy constructors/assignment for most things, with a comment as to why.